testing/tests/ui/name_mismatch_endblock.stderr

```diff
@@ -1,4 +1,5 @@
-error: problems parsing template source at row 1, column 27 near:
+error: error: expected name `foo` in `endblock` tag, found `not_foo`
```

Comment:
Don't love the `error: error:` here, can we avoid that?

Reply:
Ah yes, completely missed that. I'll remove the extra `error:` I added.